src/storage/disk_table.cc

```diff
@@ -1135,7 +1326,7 @@ int DiskTable::GetCount(uint32_t index, const std::string& pk, uint64_t& count)
     ro.snapshot = snapshot;
     // ro.prefix_same_as_start = true;
     ro.pin_data = true;
-    rocksdb::Iterator* it = db_->NewIterator(ro, cf_hs_[inner_pos + 1]);
+    rocksdb::Iterator* it = db_->NewIterator(ro, cf_hs_->at(inner_pos + 1));
```

Comment:
like here. `cf_hs = std::atomic_load_explicit(&cf_hs_, std::memory_order_acquire)`. And then use `cf_hs->at`